cni/pkg/ebpf/server/redirectionServer_linux.go

```diff
@@ -567,7 +567,7 @@ func (r *RedirectServer) attachTC(namespace string, ifindex uint32, direction st
 				},
 			},
 		}
-		if err := rtnl.Filter().Add(&filterIngress); err != nil && !errors.Is(err, os.ErrExist) {
```

Comment:
Am I correct that `Add` would be more or less identical to `Replace` as long as the `ifidx` and `fd` are the same?

As they would be today, since we don't load SxS prog versions on the same node.

Reply:
https://github.com/torvalds/linux/blob/ba0ad6ed89fd5dada3b7b65ef2b08e95d449d4ab/net/sched/cls_api.c#L2291

We could just literally interpret `Add` and `Replace`, `Add` means create a new one if nothing already existed, otherwise return `os.ErrExist`. `Replace` means create a new one if nothing already existed, otherwise replace it with the new one.